CONTRIBUTING.md

```diff
@@ -0,0 +1,149 @@
+
+* Ensure that the bug was not already reported by [searching under Issues].
+
+* If you're unable to find an (open) issue addressing the problem, [open a new
+  one]. Be sure to prefix the issue title with **[BUG]** and to include:
+
+  - a *clear* description,
+  - as much relevant information as possible, and
+  - a *code sample* or an (executable) *test case* demonstrating the expected behaviour that is not occurring.
+
+* If possible, use the following [template to report a bug].
+
+
+## Highlevel overview of how we work
+
+### Github projects
+
+We use the [Local EGA](https://github.com/orgs/EGA-archive/projects/3) github
+project in the EGA-Archive to manage our work.
+
+### How we use issues
+
+Issues are the central area for all work.
+
+### Standups
+
+Every day at 10:30 we have standups in Zoom meeting: <id>
+
+### Sprint Reviews and Planning
+
+Approximately every 2 weeks we have sprint reviews and then also a sprint
+planning meeting.
+
+
+
+## How to work on a new feauture/bug
+
+Create an issue on Github, and talk to the team members on the NBIS local-ega
+Slack channel. You can alternatively pick one already created.
+
+Assign yourself to that issue.
+
+Discussions on how to proceed about that issue take place in the comment
+section on that issue, beforehand.
+
+The keyword here is *beforehand*. It is usually a good idea to talk about it
+first. Somebody might have already some pieces in place, we avoid unnecessary
+work duplication and a waste of time and effort.
+
+
+## How we work with Git
+
+All work take place in feature branches. Give your branch a short descriptive
+name and prefix the name with:
+
+   * `feature/` if it is a code feature
+   * `hotfix/` if you are fixing an urgent bug
+
+Use comments in your code, choose variable and function names that clearly show
+what you intend to implement.
+
+Once the feature is done it is merged back into `master` by making a Pull
+Request.
+
+
+### General stuff about git and commit messages
+
+In general it is better to commit often. Small commits are easier to roll back
+and also makes the code easier to review.
+
+Write helpful commit messages that describes the changes and possibly why they
+were necessary.
+
+Each commit should contain changes that are functionally connected and/or
+related. If you for example want to write _and_ in a commit message this is a
+good sign that it should have been two commits.
+
+Learn how to select chunkgs of changed files to do multiple separate commits of
+unrelated things. This can be done with either `git add -p` or `git commit -p`.
+
+
+#### Helpfull commit messages
+
+The commit messages may be seen as meta-comments on the code that are
+incredibly helpful for anyone who wants to know how this piece of software is
+working, including colleagues (current and future) and external users.
+
+Some tips about writing helpful commit messages:
+
+#. Separate subject (the first line of the message) from body with a blank line.
```

Comment:
On https://github.com/EGA-archive/LocalEGA/blob/feature/git-guidelines/CONTRIBUTING.md I see all these messages on the same line but I am not sure if readthedocs markdown is different from github.

Reply:
That's a bug. Will fix this.